packages/web-components/fast-components/src/dialog/dialog.styles.ts

```diff
@@ -37,7 +37,7 @@ export const DialogStyles = css`
     .root {
         ${elevation} margin-top: auto;
         margin-bottom: auto;
-        border-radius: calc(var(--elevated-corner-radius));
+        border-radius: calc(var(--corner-radius));
```

Comment:
Just an interesting note...if you wanted to share styles in the future, variables can fall back to other variables. So, you could do something like this:

`border-radius: var(--elevated-corner-radius, var(--corner-radius));`

If the elevated radius is available it will use that, otherwise it will fall back to normal corner radius.

QQ: Why is `calc` needed here?

Reply:
On #1, there's no concept in the FAST of elevated corner radius, so we don't actually want to include that in the design system. On #2, I don't think it's needed, I just dumped the var. I can take a look at this though.

Reply:
Ah, #2 is because it's supposed to be multiplying by 1px

Reply:
Make sense. I wondered if it needed the calc to apply a unit.